apps/prairielearn/src/tests/exampleCourseQuestionsComplete.test.ts

```
@@ -0,0 +1,370 @@
+  // Extremely large question
+  'element/dataframe',
+];
+
+describe('Internally Graded Question Lifecycle Tests', function () {
+  this.timeout(60000);
+  const originalProcessQuestionsInServer = config.features['process-questions-in-server'];
+
+  before('set up testing server', helperServer.before());
+  before('enable process-questions-in-server', () => {
+    config.features['process-questions-in-server'] = true;
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow before duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)